code/modules/asset_cache/assets/paper.dm

```diff
@@ -16,5 +16,9 @@
 		"stamp-chap" = 'icons/stamp_icons/large_stamp-chap.png',
 		"stamp-mime" = 'icons/stamp_icons/large_stamp-mime.png',
 		"stamp-centcom" = 'icons/stamp_icons/large_stamp-centcom.png',
-		"stamp-syndicate" = 'icons/stamp_icons/large_stamp-syndicate.png'
+		// SKYRAT EDIT START - CUSTOM STAMPS
+		"stamp-syndicate" = 'icons/stamp_icons/large_stamp-syndicate.png',
```

Comment:
Is the syndicate stamp actually a Skyrat edit? Feels like a mistake.

Reply:
this one was a goof, will fix later